pack/fhv/fhvc.json

```diff
@@ -716,10 +716,233 @@
+    "illustrator": "Mauro Dal Bo",
+    "name": "Lost Self",
+    "pack_code": "fhvc",
+    "position": 156,
+    "quantity": 1,
+    "stage": 2,
+    "text": "[reaction] After you succeed at a skill test at a [[Lair]] location during your turn: Reveal cards from the bottom of The Abyss for each point you succeed by. Draw 1 of them, shuffle the rest, and place them on top of The Abyss.\n[fast] Spend 1 clue: You get +2 skill value for this skill test.\n<b>Objective</b> - At the end of the round, if there are 3 or more revealed [[Cave]] locations in play with no clues on them, advance.",
+    "type_code": "act"
+  },
+  {
+    "back_flavor": "You stand next to Dr. Marquez at the base of a mound of desiccated bodies. Above, the Abyss flickers in the still, cold air. A hazy humanoid shape reaches toward you as the brilliance flares again, blindingly bright. \"It's no good. We need to disrupt it. Feed it something it can't 'digest',\" Dr. Marquez says soberly. She hands you her satchel filled with survey notes and samples. \"I want you to have this.\" You ask why. \"Someone needs to tell Dr. Armitage the story of this...\" her eyes narrow, staring up at the flickering iridescence.\" ...this colour out of space.\"",
+    "back_name": "Ad Nihilum",
+    "back_text": "The investigators must decide <i>(choose one)</i>:\n- Let Dr. Marquez carry out her plan <b>(→R1)</b> <i>(page 66)</i>.\n- Do it ourselves <b>(→R2)</b>. <i>(page 67)</i>. (You may only choose this if an investigator controls Prismatic Shard story asset.)",
+    "clues": 2,
+    "code": "10657",
+    "double_sided": true,
+    "encounter_code": "fate_of_the_vale",
+    "encounter_position": 7,
+    "faction_code": "mythos",
+    "flavor": "Dr. Marquez needs your help to save the Vale!",
+    "illustrator": "Mauro Dal Bo",
+    "name": "Fate of the Vale (v.I)",
+    "pack_code": "fhvc",
+    "position": 157,
+    "quantity": 1,
+    "stage": 3,
+    "text": "[reaction] After you succeed at a skill test at a [[Lair]] location during your turn: Reveal cards from the bottom of The Abyss for each point you succeed by. Draw each single-sided enemy card revealed by this effect. Remove each other revealed card from the game.\n[fast] Spend 1 clue: You get +2 skill value for this skill test.\n<b>Objective</b> - At the end of the round, if each surviving investigator is at The Abyss, investigators may spend the requisite clues, as a group, to advance.",
+    "type_code": "act"
+  },
+  {
+    "back_flavor": "\"Get in the truck!\" A resident waves a crowd of dazed locals onto the vehicle. The remaining survivors pile into trucks and weather-beaten bikes and drive down the winding southern road. As you pass the dilapidated harbor, a pillar of pure light shoots up from the village center followed by an oily, viscous cloud. The train of survivors have just taken refuge in the southern lighthouse when a shockwave of hot air tears across the island with a hurricane force.",
```

Comment:
```suggestion
    "back_flavor": "\"Get in the truck!\" A resident waves a crowd of dazed locals onto the vehicle. The remaining survivors pile into trucks and weatherbeaten bikes and drive down the winding southern road. As you pass the dilapidated harbor, a pillar of pure light shoots up from the village center followed by an oily, viscous cloud. The train of survivors have just taken refuge in the southern lighthouse when a shockwave of hot air tears across the island with a hurricane force.",
```

Reply:
This is a typo in the original text, `weather-beaten` is the correct spelling.